src/features/channels/components/channel-list/search-channel/SearchChannel.tsx

```diff
@@ -69,14 +92,35 @@ const ChannelList = () => {
                 <ListItemText primary="Wallet" />
               </ListItemButton>
             </ListItem>
+
+            <Modal open={open} onClose={closeModal} title="Hidden Channels">
+              <Card sx={{ minWidth: 400 }}>
+                {hiddenChannel.map(
+                  (channel) =>
+                    !channel.visible && (
+                      <Box>
+                        <Divider />
+                        <Box className={styles.cardContainer}>
```

Comment:
Remember that when iterating through and array and rendering the items of that array you need to specify a 'key' prop in the parent component of what you're returning in the 'map' function. If you look at the console right now you'll get a nasty error saying you're missing the 'key' prop.

Reply:
ok, thank you. I will make the changes